jujugui/static/gui/src/app/components/deployment-flow/login/_login.scss

```diff
@@ -0,0 +1,100 @@
+.deployment-login {
+  &__intro {
+    margin-top: 20px;
+  }
+
+  &__features {
+    margin-top: 20px;
+  }
+
+  &__feature {
+    margin-bottom: 20px;
+  }
+
+  &__signup {
+    margin-top: 20px;
+    text-align: center;
+
+    .usso-login {
+      display: inline;
+      margin-left: 0.5em;
+    }
+  }
+
+  &__tick {
+    margin-right: 1em;
+  }
+
+  &-signup {
+    a {
+      color: $link-blue;
+    }
+
+    .deployment-section__content {
+      background-color: transparent;
+    }
+
+    .usso-login,
+    .usso-login__action {
+      display: inline-block;
+    }
+
+    .usso-login {
+      margin-right: 10px;
+    }
+
+    .usso-login__action {
+      width: auto;
+    }
+
+    &__message {
+      margin-right: 20px;
+      font-size: 20px;
+    }
+  }
+
+  &-features {
+    margin-bottom: 0;
+    background-color: $dark-header;
+    color: $white;
+
+    &__logo {
```

Comment:
Can you add a :first-item {margin-top: -12px} to this to align the copy to the left to the middle of these logos while maintaining the correct padding for the text.